gpu/types.go

```diff
@@ -81,8 +83,8 @@ func (l GpuInfoList) ByLibrary() []GpuInfoList {
 	for _, info := range l {
 		found := false
 		requested := info.Library
-		if info.Variant != CPUCapabilityNone {
-			requested += "_" + info.Variant.String()
+		if info.Variant != CPUCapabilityNone.String() {
+			requested += "_" + info.Variant
```

Comment:
Hi @dhiltgen ,

I'm sorry for suddenly jumping into this PR, but I was testing this change in my env and hit some issue where multiple GPUs are no longer used for some reason. 

I'm wondering line 97 needs to be changed as well. Otherwise `libs` contains `info.Library`s, and it is never equal to `requested`, which can be `info.Library + "_" + info.Variant`?